examples/Example/src/new_api/chat_heads/index.tsx

```diff
@@ -0,0 +1,193 @@
+import React, { useState } from 'react';
+import { StyleSheet, ImageStyle, LayoutChangeEvent } from 'react-native';
+import { Gesture, GestureDetector } from 'react-native-gesture-handler';
+import Animated, {
+  useAnimatedReaction,
+  useAnimatedStyle,
+  useSharedValue,
+  withSpring,
+} from 'react-native-reanimated';
+import { SafeAreaView } from 'react-native-safe-area-context';
+
+const HEADER_HEIGHT = 44; // let's assume it's always the same for ease of understanding ;)
+const CHAT_HEADS = [
+  { imageUrl: 'https://avatars0.githubusercontent.com/u/379606?v=4&s=460' },
+  { imageUrl: 'https://avatars.githubusercontent.com/u/15989228?v=4&s=460' },
+  { imageUrl: 'https://avatars3.githubusercontent.com/u/90494?v=4&s=460' },
+  { imageUrl: 'https://avatars3.githubusercontent.com/u/726445?v=4&s=460' },
+];
+
+interface AnimatedOffset {
+  x: Animated.SharedValue<number>;
+  y: Animated.SharedValue<number>;
+}
+
+interface FollowingChatHeadProps {
+  imageUri: string;
+  offset: AnimatedOffset;
+  offsetToFollow: AnimatedOffset;
+  style?: ImageStyle;
+}
+
+function FollowingChatHead({
+  imageUri,
+  style,
+  offset,
+  offsetToFollow,
+}: FollowingChatHeadProps) {
+  useAnimatedReaction(
+    () => {
+      return {
+        x: offsetToFollow.x.value,
+        y: offsetToFollow.y.value,
+      };
+    },
+    (followOffset) => {
+      offset.x.value = withSpring(followOffset.x);
+      offset.y.value = withSpring(followOffset.y);
+    }
+  );
```

Comment:
Why `useAnimatedReaction` instead of `useDerivedValue`?

Reply:
Yeah I forgot about it, thanks!